backends/aerospike.go

```diff
@@ -45,12 +45,26 @@ type AerospikeBackend struct {
 }
 
 func NewAerospikeBackend(cfg config.Aerospike, metrics *metrics.Metrics) *AerospikeBackend {
-	client, err := as.NewClient(cfg.Host, cfg.Port)
+	var hosts []*as.Host
+
+	clientPolicy := as.NewClientPolicy()
+	if len(cfg.User) > 0 {
+		clientPolicy.User = cfg.User
+	}
+	if len(cfg.Password) > 0 {
+		clientPolicy.Password = cfg.Password
+	}
```

Comment:
There's no need to check for empty strings. Since the `clientPolicy` defaults `User` and `Password` to an empty string. there's no harm in assigning it to an empty string.

Reply:
I'll make this change.